text/0000-make-cargo-install-extensible.md

```diff
@@ -0,0 +1,122 @@
+# RFC: Make cargo install extensible
+
+- Feature Name: extensible_cargo_install
+- Start Date: 2018-03-27
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+## Summary
+
+The goal of this RFC is to introduce the ability for an end user to be able to extend the `cargo install` command arbitrarily to include instructions that should be executed occur after `cargo install <project>`. 
```

Comment:
"to be able" seems redundant here?
I propose the following wording:
> This RFC gives end users the ability to extend the `cargo install` command arbitrarily, allowing them to include instructions executed after `cargo install <project>`.

Reply:
Sure thing. Happy to update.